examples/atari/reproduction/rainbow/README.md

```diff
@@ -21,6 +21,21 @@ python train_rainbow.py [options]
 
 To view the full list of options, either view the code or run the example with the `--help` option.
 
+## Running our Pretrained models
+
+We have already pretrained models from this script for all the domains list in the [results](#Results) section. To load a pretrained model:
+
+```
+python train_rainbow.py --demo --load-pretrained --env BreakoutNoFrameskip-v4 --pretrained-type best --gpu -1
+```
+
+### Useful Options
+- `--gpu`. Specifies the GPU. If you do not have a GPU on your machine, run the example with the option `--gpu -1`.
+- `--env`. Specifies the environment. 
+- `--demo`. Runs an evaluation, instead of training the agent.
+- `--load-pretrained` Loads the pretrained model. Both `--load` and `--load-pretrained` cannot be used together.
+- `--pretrained-type`. Either `best` (the best intermediate network during training) or `final` (the final network after training).
+
```

Comment:
I think `## Running the Example` and `## Running our Pretrained models` should be merged (not only for this example but also for other examples). Having two `### Useful Options`s that largely overlap is not good.

Reply:
Ok, will change.